plugin/import-js.vim

```diff
@@ -1,3 +1,13 @@
 command! ImportJSWord call importjs#Word()
 command! ImportJSGoto call importjs#Goto()
 command! ImportJSFix call importjs#Fix()
+
+"call it when entering vim in order to have no lag at all
+call importjs#FindPackageJson(getcwd())
+
+" call on cursorhold since package.json can be created
+" after vim was started
+augroup loadImportJsOnHold
+  au!
+  au CursorHold,CursorHoldI * call importjs#FindPackageJson(getcwd())
+augroup END
```

Comment:
I think there is some useful stuff in here, but I'm going to need more context in order to review this the right way? Can you explain why you're making this change, what different options you considered, and why you ended up with the solution you have here? This should preferably go in the commit message so that others (including future you) can understand why the change was made in the future.